pkg/container-disk/container-disk.go

```diff
@@ -368,7 +354,7 @@ func CreateEphemeralImages(
 	disksInfo map[string]*disk.DiskInfo,
 ) error {
 	// The domain is setup to use the COW image instead of the base image. What we have
-	// to do here is only create the image where the domain expects it (GetDiskTargetPartFromLauncherView)
+	// to do here is only create the image where the domain expects it (GetDiskTargetFromLauncherView)
```

Comment:
You mean `GetDiskTargetPathFromLauncherView`, not `GetDiskTargetFromLauncherView` right?
You mention `GetDiskTargetFromLauncherView` also in the commit desc but it's not what is changed in the commit

Reply:
Yes, thanks!